### ISSUE TITLE
Update Documentation

### ISSUE THREAD

**Issue by H4RSH-K** [H4RSH-K/the-boring-school#3]
Make documentation better!

**Comment by omriattiya** [comment]
can I help with this?

**Comment by H4RSH-K** [comment]
Assigned to you! Anyways this issue is open for all

**Comment by sb2356-iiitr** [comment]
So we need to edit the readme file (if required), right?

**Comment by H4RSH-K** [comment]
Yes, just do not change the core principle behind it 😅 sure you can work on it (if you feel like it)

**Comment by sb2356-iiitr** [comment]
> Yes, just do not change the core principle behind it  sure you can work on it (if you feel like it)

Ok
